apps/documentation/src/app/pages/components/chart-legend/demos/legend/chart-legend-demo.component.ts

```diff
@@ -1,9 +1,17 @@
+import { CommonModule } from '@angular/common';
 import { Component, OnInit } from '@angular/core';
-import { DataLayerManagerService } from '@elimuinformatics/ngx-charts-on-fhir';
+import {
+  DataLayerManagerService,
+  FhirChartLegendModule,
+  FhirChartTagsService,
+} from '@elimuinformatics/ngx-charts-on-fhir';
 
 @Component({
   selector: 'example-chart-legend-demo',
   templateUrl: './chart-legend-demo.component.html',
+  standalone: true,
+  imports: [CommonModule, FhirChartLegendModule],
+  providers: [FhirChartTagsService],
```

Comment:
does tags service need to be provided in the components? I don't think we do that in real apps.